ej2-angular/check-box/how-to/name-and-value-in-form-submit.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Name and value in form submit in Angular Check box component | Syncfusion
-description: Learn here all about Name and value in form submit in Syncfusion Angular Check box component of Syncfusion Essential JS 2 and more.
+title: Name and value in form submit in Angular CheckBox Component | Syncfusion
```

Comment:
Check control name casing in all places.

Reply:
changed